codewof/templates/users/achievements.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block title %}Achievements{% endblock %}
+
+{% block content %}
+  <div class="container">
+    <h1 class="display-3 orange-underline text-center">Achievements</h1>
+    <h3>Achievements earned: {{num_badges_earned}}/{{num_badges}}</h3>
+    <div class="col-12 col-md-8 offset-md-2 mb-3">
+        <br>
+        <div class="row">
+            {% for badge in user.profile.earned_badges.all %}
+                <div class="col-4 badge-container">
+                    <img class="badge-icon" src="{% static badge.icon_name %}">
+                    <b>{{badge.display_name}}</b>
```

Comment:
for consistency add a space before/after inside the curly braces (also on line 9)